src/screens/FeedbackScreen.tsx

```diff
@@ -0,0 +1,219 @@
+          </View>
+        </View>
+      </Modal>
+            <TouchableOpacity style={styles.button} disabled={text == "" || selection == ""} onPress={() => setModalVisible(!modalVisible)}>
+                <Text style={styles.button_text}>
+                    SUBMIT FEEDBACK
+                </Text>
+            </TouchableOpacity>
+        </ScrollView>
+    )
+};
+
+type EmojiProp = {
+  emojitype  : {emoji : string , text : string},
+  containerstyle : ViewStyle | ViewStyle[]
+}
+
+const Emojis = (props : EmojiProp) => {
+  const text : string = props.emojitype.text
+  const emoji : string = props.emojitype.emoji
+  return (
+    <View style={props.containerstyle}>
+      <Text style={styles.emoji_image}> {emoji} </Text>
+      <Text style={styles.emoji_text}>{text}</Text>
+    </View>
+    )
+}
+
+type FeedbackStyle = {
+  main_container : ViewStyle,
+  feedback_text : TextStyle,
+  response_container : ViewStyle,
+  emoji_container : ViewStyle,
+  clicked_emoji_container : ViewStyle,
+  response_container_text : TextStyle,
+  emojis_response_container : ViewStyle,
+  textinput_text : TextStyle,
+  button : ViewStyle,
+  button_text : TextStyle,
+  modal_view : ViewStyle,
+  modal_button : ViewStyle,
+  modal_text : TextStyle,
+  modal_image : ImageStyle,
+  emoji_text : TextStyle,
+  emoji_image : TextStyle,
+  modal_overlay : ViewStyle
+}
+
+const styles = StyleSheet.create<FeedbackStyle>({
+    main_container : {
+        flex : 1 ,
+        alignItems : "center",
+        backgroundColor : colors.DROIDCONKE_WHITE
+    },
+    feedback_text : {
+        textAlign : "center",
+        color : colors.DROIDCONKE_BLUE ,
+        width : "80%",
+        margin : 30,
+        fontSize : 18,
+        fontFamily : fonts.MONTSERRAT_BOLD
+    },
+    response_container : {
+        alignItems : "center",
+        marginBottom : 30,
+        borderColor : "#F5F5F5",
+        borderWidth : 1,
+        borderRadius : 8,
+        width : "90%"
+    },
+    emoji_container : {
+        height : 67,
+        width : 67,
+        backgroundColor : colors.DROIDCONKE_PEARL,
+        borderRadius : 4 ,
+        margin : 30 ,
+        justifyContent : "center",
+        alignContent : "center",
+    },
+    clicked_emoji_container : {
+      opacity : 0.2
+    },
+    response_container_text : {
+        color : colors.DROIDCONKE_BLACK,
+        fontSize : 14 ,
+        fontFamily : fonts.MONTSERRAT_REGULAR,
+        textAlign : "center",
+        marginTop : 17,
+    },
+    emojis_response_container : {
+        flexDirection : "row",
+        justifyContent : "space-around"
+    },
+    textinput_text : {
+        backgroundColor : colors.DROIDCONKE_PEARL,
+        width : "90%",
+        marginBottom : 30,
+        height : 115,
+        textAlign : "left",
+        textAlignVertical : "top",
+        padding : 5,
+        borderRadius : 7,
+        fontFamily : fonts.MONTSERRAT_LIGHT,
+        fontSize : 14,
+        color : "#C3C3C3"
```

Comment:
Uhm could you define this color `#C3C3C3` on the color constants file instead?
You can name it in a descriptive way.

Reply:
> Uhm could you define this colour ` #C3C3C3 ` on the colour constants file instead? You can name it in a simple descriptive way.

Reply:
hehe
IDK, maybe DROIDCONKE_FEEDBACK_INPUT or something shorter. 
I leave that up to you!